packages/grid/x-grid/src/tests/apiRef.XGrid.test.tsx

```diff
@@ -154,7 +154,13 @@ describe('<XGrid /> - apiRef', () => {
 
   it('getDataAsCsv should return the correct string representation of the grid data', () => {
     render(<TestCase />);
-
     expect(apiRef.current.getDataAsCsv()).to.equal('Brand\r\nNike\r\nAdidas\r\nPuma');
+    apiRef.current.updateRows([
+      {
+        id: 1,
+        brand: 'Adidas,Reebok',
```

Comment:
could we had a test with `brand: 'A "d,i,das '`

Reply:
done